features/api-token.feature

```diff
@@ -13,14 +13,12 @@ Feature: User API Token
         - Users can revoke a token at any time
         - No one can ever look up the raw value after initially generating the token
 
-    Background:
```

Comment:
Moved authentication to a `before` hook, since other `before` hooks need to be authenticated.